routes/utils.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function(connection) {
+
+    /* Middleware to check whether the user is logged in */
+    function isAuthenticated(req, res, next) {
+        if(req.session.authenticated) {
+            return next();
+        } else {
+            res.redirect("/login");
+        }
+    };
+    
+    /* Middleware to check whether the user is logged in */
+    function isAuthorized(req, res, next) {
```

Comment:
The method name could be misleading. For example, a user may be "authorized" to create or update a marker. Since this method is actually checking to see if the user is an admin, I'd suggest renaming it to `isAdmin(...)` or `isAuthenticatedAdmin(...)`

Reply:
Pushed the change :)